deploy/gk-deploy

```diff
@@ -20,13 +20,15 @@ LOG_FILE=''
 VERBOSE=0
 CLI=''
 GLUSTER=0
-TEMPLATES="templates"
+KUBE_TEMPLATES_DEFAULT="./kube-templates"
+OCP_TEMPLATES_DEFAULT="./ocp-templates"
+TEMPLATES="openshift"
```

Comment:
That's a dev shortcut I forgot to remove. 'openshift' was the name of the directory I was using for templates, so I hard-coded it cause I was lazy. :)